spiketools/stats/shuffle.py

```diff
@@ -199,6 +199,10 @@ def shuffle_circular(spikes, shuffle_min=20000, n_shuffles=1000, random_state=No
     -------
     shuffled_spikes : 2d array
         Shuffled spike times.
+		
+    Notes
+    -----
+    The input shuffle_min should always be less than the number of spikes (of input spikes).
```

Comment:
I'm not sure I totally follow this comment. I think maybe the value of `shuffle_min` should be less than the duration of the spike train (for example, if there are 100 spikes covering time range 0 to 10000 ms, should shuffle_min be less than 10000? I don't think it should be that the value of shuffle_min needs to be less than 100 though). Could you rephrase this?

Reply:
You're right, I did not describe it correctly! I changed the wording, please let me know if this is clearer, or if I should work a little more on it.